test/e2e/memcached_test.go

```diff
@@ -752,19 +737,14 @@ func memcachedOperatorMetricsTest(t *testing.T, f *framework.Framework, ctx *fra
 	return nil
 }
 
-func getMetrics(t *testing.T, f *framework.Framework, label map[string]string, ns, port string) ([]byte, error) {
```

Comment:
There is a TODO left by me on line 671. We can remove the 1 now and instead fail when there are more than 0 problems detected. As this was fixed in 1.14 k8s upstream. Thanks!

> Change to 0, when we upgrade to 1.14.

Reply:
If you want to I can do it in a separate PR?

p.s. thanks for the label -> labels change :) <3

Reply:
Awesome. Yes a follow-up PR would be great.